toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1560,6 +1567,21 @@ func setGrubCfgBootUUID(bootUUID, grubPath string) (err error) {
 	return
 }
 
+func setGrubCfgBootPrefix(bootPrefix, grubPath string) (err error) {
+	const (
+		bootPrefixPattern = "{{.BootPrefix}}"
+	)
+	var cmdline configuration.KernelCommandLine
+
+	logger.Log.Debugf("Adding BootPrefix('%s') to %s", bootPrefix, grubPath)
```

Comment:
>%s [](start = 47, length = 2)

Nit: I'd also enclose the path string in single quotes/brackets. It will make debugging easier by making it clear where there should still be some content when due to some error `grubPath` ends up being empty.

Reply:
Fixed across file